docs/development/testing_and_dependencies.md

```diff
@@ -14,6 +14,14 @@ make test-cov
 open gardener.coverage.html
 make test-cov-clean
 ```
+### sigs.k8s.io/controller-runtime env test
+
+Some of the integration tests in Gardener are using the `sigs.k8s.io/controller-runtime/pkg/envtest` package.
+It sets up a temporary control plane (etcd + kube-apiserver) against the integration tests can run.
+The `test` and `test-cov` rules in the `Makefile` do prepare this env test automatically by downloading the respective binaries (if not yet present) and setting the necessary environment variables.
```

Comment:
The release note of the PR mentions also for `make test-integration-extensions-controllers` but it is not added atm. Do you want to add it or it is an outdated comment?